### DISCUSSION TITLE
Cursor changes when dragging an item

### DISCUSSION THREAD

**Discussion by fin4zik** [peter-kish/gloot#224] [Q&A]
Hello,
I selected a custom image for cursor in project settings, but when I drag an item, the cursor image changes. Is there any way to disable this? Video demonstration:
https://github.com/user-attachments/assets/175e8d0b-0099-4492-9543-f586adf51562

**Comment by peter-kish** [comment]
As described in the [Godot docs](https://docs.godotengine.org/en/stable/tutorials/inputs/custom_mouse_cursor.html), you can define custom mouse cursors for each cursor "shape" with [Input.set_custom_mouse_cursor](https://docs.godotengine.org/en/stable/classes/class_input.html#class-input-method-set-custom-mouse-cursor) (`CURSOR_DRAG` for example). It seems like the project settings only affect the standard cursor shape (`CURSOR_ARROW`).
